tests/http/bad-actors.test.js

```diff
@@ -1,16 +0,0 @@
-var server = require('../../server');
```

Comment:
Please do a `curl` test locally to see if it redirects properly and if so then this is r+